tests/drivers/spi/spi_loopback/boards/numaker_pfm_m467.overlay

```diff
@@ -0,0 +1,34 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&pinctrl {
+/* CS/CLK/MISO/MOSI --> EVB's NU5: SS/CLK/MISO/MOSI --> PA11/PA10/PA9/PA8 */
+	spi2_default: spi2_default {
+		group0 {
+			pinmux = <PA11MFP_SPI2_SS>,
+				<PA10MFP_SPI2_CLK>,
+				<PA9MFP_SPI2_MISO>,
+				<PA8MFP_SPI2_MOSI>;
+		};
+	};
+};
+
+&gpioa {
+	status = "okay";
+};
+
+&spi2 {
+	slow@0 {
+		compatible = "test-spi-loopback-slow";
+		reg = <0>;
+		spi-max-frequency = <500000>;
+	};
+	fast@0 {
+		compatible = "test-spi-loopback-fast";
+		reg = <0>;
+		spi-max-frequency = <16000000>;
+	};
+	status = "okay";
+	pinctrl-0 = <&spi2_default>;
+	pinctrl-names = "default";
+/*    cs-gpios = <&gpioa 11 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>; */
```

Comment:
Drop it?

Reply:
If it's for the loopback test, why not just enabling it? There's plenty of tests like this that just picks a pin that makes sense so it's ready to use.

Reply:
Agree with your opinion to drop this comment.